yarn.lock

```diff
@@ -16703,7 +16703,7 @@ shelljs@0.5.3:
   resolved "https://registry.yarnpkg.com/shelljs/-/shelljs-0.5.3.tgz#c54982b996c76ef0c1e6b59fbdc5825f5b713113"
   integrity sha1-xUmCuZbHbvDB5rWfvcWCX1txMRM=
 
-shelljs@^0.8.2:
```

Comment:
Members of the monorepo should not have their own `yarn.lock` files. This one can simply be deleted. After doing so, run `yarn reboot:full` in the root of the monorepo and `yarn.lock` there will pick up the needed changes.

Reply:
This is the global yarn.lock though. Is it not ok?

Reply:
Whoops, you're totally correct. I thought I was looking at a file in `packages/embark-solc/`.